app/controllers/concerns/hydranorth/files_controller_behavior.rb

```diff
@@ -27,18 +41,9 @@ def attributes
       attributes = params
     end
 
+
     def presenter
-      if @generic_file[:resource_type].include? Sufia.config.special_types['cstr']
-        Hydranorth::CstrPresenter.new(@generic_file)
-      elsif @generic_file[:resource_type].include? Sufia.config.special_types['ser']
-        Hydranorth::SerPresenter.new(@generic_file)
-      elsif @generic_file[:remote_resource] == "dataverse"
-        Hydranorth::DataversePresenter.new(@generic_file)
-      elsif @generic_file[:resource_type].include? Sufia.config.special_types['thesis']
-        Hydranorth::ThesisPresenter.new(@generic_file)
-      else
-        Hydranorth::GenericFilePresenter.new(@generic_file)
-      end
+      presenter_class.new(@generic_file)
```

Comment:
This is how it's already defined in sufia, so we don't have to override presenter at all anymore https://github.com/projecthydra/sufia/blob/6.x-stable/app/controllers/concerns/sufia/files_controller_behavior.rb#L171